yb_migrate/src/srcdb/mysql.go

```diff
@@ -111,3 +112,15 @@ func (ms *MySQL) ExportSchema(exportDir string) {
 func (ms *MySQL) ExportData(ctx context.Context, exportDir string, tableList []string, quitChan chan bool, exportDataStart chan bool) {
 	ora2pgExportDataOffline(ctx, ms.source, exportDir, tableList, quitChan, exportDataStart)
 }
+
+func (ms *MySQL) ExportDataPostProcessing(exportDir string, tablesProgressMetadata *map[string]*utils.TableProgressMetadata) {
```

Comment:
No need to take a pointer to a map. Just pass a map directory--it is already a reference type.